www/js/components/add-point-hours.js

```diff
@@ -16,14 +12,19 @@ export class AddPointHours extends Component {
   constructor(props) {
     super(props);
     this.state = {
```

Comment:
I don't think we can make this stateless, unless we want to add an intermittent state for an hours being added (besides, this logic might change with the appropriate new UI).